pos_product_mergeable_line/static/src/js/db.js

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright (C) 2023-Today: GRAP (<http://www.grap.coop/>)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define("pos_product_mergeable_line.db", function (require) {
+    "use strict";
+
+    var models = require("point_of_sale.models");
+
+    models.load_fields("product.product", ["pos_mergeable_line"]);
```

Comment:
IMO this line should be moved to `pos_product_mergeable_line/static/src/js/models.js`, and you can drop the file.

The name `pos_product_mergeable_line.db` suggests overrides on `point_of_sale.db`, but it's just "extending" models.js

Reply:
Ow thanks ! Totally right.
Note : Oca modules requires some clean regarding that topic.

- https://github.com/OCA/pos/blob/12.0/pos_tare/static/src/js/db.js#L12
- https://github.com/OCA/pos/blob/12.0/pos_multi_ean/static/src/js/db.js#L7
- https://github.com/OCA/pos/blob/12.0/pos_supplierinfo_barcode/static/src/js/db.js#L7
- https://github.com/OCA/pos/blob/12.0/pos_supplierinfo_search/static/src/js/db.js#L7